### ISSUE TITLE
lxpanel "Minimize all" affects all desktop pages/workspaces when iconifying - but only current when shading

### ISSUE THREAD

**Issue by db0451** [lxde/lxqt#805]
Hi, firstly, thanks for a great desktop experience!

One fairly minor thing I've noticed: Using the standard applet "Minimize All Windows" to iconify affects all windows on both desktops/workspaces. However, using the same applet to shade only affects windows located on the currently active desktop.

I feel these 2 behaviours - accessed by different clicks on the same icon - should behave consistently across multiple desktops. I suggest that they should both affect windows on the active page only - or offer an option to affect all pages.

I've researched a bit but am not aware of any way to edit what this applet does - sorry if I've just missed it, but maybe it would still be a beneficial amendment to the default behaviour.

OS is latest Debian 8/Jessie, x64, LXDE CD1 edition (which uses OpenBox as WM and PCManFileManager as desktop background).

Many thanks!
Daniel

**Comment by palinek** [comment]
> OS is latest Debian 8/Jessie, x64, LXDE CD1 edition (which uses OpenBox as WM and PCManFileManager as desktop background).

Are you talking about LXDE & lxpanel or LXQt & lxqt-panel ?

**Comment by db0451** [comment]
Sorry, I had misread the Wikipedia article and thought that LXDE was fully
migrated to Qt; I didn't realise the two versions exist in parallel.

To confirm, it's for the normal (GTK+?) version.

Apologies and if I need to repost this anywhere else, let me know.

thanks
Daniel

**Comment by palinek** [comment]
Yes, this issue needs to be reported elsewhere....

@jleclanche where are LXDE issues reported?

**Comment by jleclanche** [comment]
LXDE issues can be reported here. The LXDE tracker is pretty much nonexistant right now.

**Comment by db0451** [comment]
Thanks all. Do I need to open another ticket with the same text, or can
this one be moved?

cheers
Daniel

**Comment by pmattern** [comment]
> The LXDE tracker is pretty much nonexistant right now.

Err, really? I mean @LStranger is actually still active on both the trackers of http://sourceforge.net/projects/lxde and http://sourceforge.net/projects/pcmanfm/.

**Comment by jleclanche** [comment]
We're moving away from SF. At least we're supposed to. But I have no idea what's going on with that, and honestly given the LXDE team's unresponsiveness, I'm starting to care very little.

LStranger is the only active person on the SF tracker, AFAIK. So for libfm/pcmanfm it's still good, but for lxpanel I have no idea.

**Comment by pmattern** [comment]
> We're moving away from SF. At least we're supposed to. [...] unresponsiveness [...]

I see. Yet I still think LXDE issues should get posted on sf.net:
The repos on github.com/lxde belonging to LXDE aren't suitable to provide a generic LXDE issue tracker. Opening another repo just to provide an issue tracker and not dealing with any code doesn't seem reasonable. And please let's not use the tracker on github.com/lxde/lxqt which would get pretty confusing.
So going on using the sf trackers seems the most reasonable thing to do. Any lack of activity is regrettable but not something us LXQt folks are responsible for, imo.

**Comment by jleclanche** [comment]
Some projects, including the old lxsession, moved to being tracked on github (not in this repository, in their own). Ideally all projects would move there but like I said, I'm not sure how people want this handled and I don't get answers when I ask, so I'm leaving it alone.
